code/api/src/modules/product/query.test.js

```diff
@@ -0,0 +1,54 @@
+import request from 'supertest'
+import express from 'express'
+import schema from '../../setup/schema'
+import graphqlHTTP from 'express-graphql'
+
+describe('product query', () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+
+    server.use(
+      "/", 
+      graphqlHTTP({
+        schema: schema,
+        graphiql: true
+      })
+    )
+  })
+
+  it("is true", () => {
+    expect(true).toBe(true)
+  })
```

Comment:
We can probably remove this now that we have other more substantial tests below it.